packages/core/src/Drawer/Drawer.tsx

```diff
@@ -53,13 +57,15 @@ export const Drawer: React.FC<DrawerProps> = ({
   onClose,
   onCollapse,
   placement = 'right',
+  zIndex = 1,
```

Comment:
Should the default `zIndex` be `auto` since we weren't explicitly setting it before?

Reply:
Actually by not setting zIndex it defaults to auto, hence why the default value